tests/assets/interop_summary.txt

```diff
@@ -0,0 +1,64 @@
+6              ,-              ,96             ,2355 +/- 0     ,76.01 +/- 3.42 ,0.149 / 0.069  ,0.193*c - 0.385,0.073*c + 0.449,621.21         ,472.21         ,81.78          ,70.95          ,150            ,0.56 +/- 0.03  ,0.59 +/- 0.32  ,0.15 +/- 0.12  ,0.26 +/- 0.17  ,0.31 +/- 0.18  ,162 +/- 15
+6              ,1              ,48             ,2355 +/- 0     ,76.05 +/- 3.07 ,0.148 / 0.068  ,0.191*c - 0.248,0.071*c + 0.490,310.61         ,236.21         ,81.22          ,35.50          ,-              ,0.56 +/- 0.03  ,0.57 +/- 0.32  ,0.12 +/- 0.06  ,0.23 +/- 0.15  ,0.29 +/- 0.18  ,152 +/- 11
+6              ,2              ,48             ,2355 +/- 0     ,75.98 +/- 3.78 ,0.149 / 0.071  ,0.195*c - 0.522,0.075*c + 0.408,310.61         ,235.99         ,82.34          ,35.45          ,-              ,0.57 +/- 0.03  ,0.60 +/- 0.33  ,0.17 +/- 0.16  ,0.28 +/- 0.18  ,0.33 +/- 0.19  ,172 +/- 10
+7              ,-              ,96             ,2355 +/- 0     ,75.04 +/- 3.98 ,0.150 / 0.082  ,0.193*c - 0.300,0.078*c + 0.597,621.21         ,466.14         ,80.82          ,70.07          ,150            ,0.47 +/- 0.02  ,0.68 +/- 0.42  ,0.15 +/- 0.09  ,0.26 +/- 0.15  ,0.33 +/- 0.19  ,170 +/- 15
+7              ,1              ,48             ,2355 +/- 0     ,75.21 +/- 3.48 ,0.150 / 0.080  ,0.191*c - 0.158,0.076*c + 0.654,310.61         ,233.59         ,80.49          ,35.12          ,-              ,0.47 +/- 0.02  ,0.60 +/- 0.27  ,0.15 +/- 0.07  ,0.24 +/- 0.10  ,0.30 +/- 0.13  ,159 +/- 8
+7              ,2              ,48             ,2355 +/- 0     ,74.87 +/- 4.46 ,0.150 / 0.084  ,0.196*c - 0.442,0.080*c + 0.540,310.61         ,232.54         ,81.15          ,34.94          ,-              ,0.48 +/- 0.02  ,0.75 +/- 0.53  ,0.14 +/- 0.11  ,0.28 +/- 0.19  ,0.36 +/- 0.24  ,181 +/- 12
+8              ,-              ,96             ,2355 +/- 0     ,75.32 +/- 4.10 ,0.153 / 0.073  ,0.194*c - 0.195,0.075*c + 0.501,621.21         ,467.92         ,80.42          ,70.29          ,150            ,0.51 +/- 0.02  ,0.63 +/- 0.32  ,0.18 +/- 0.23  ,0.29 +/- 0.23  ,0.35 +/- 0.23  ,163 +/- 11
+8              ,1              ,48             ,2355 +/- 0     ,76.40 +/- 3.60 ,0.155 / 0.074  ,0.191*c - 0.013,0.074*c + 0.572,310.61         ,237.31         ,80.56          ,35.65          ,-              ,0.51 +/- 0.02  ,0.53 +/- 0.23  ,0.13 +/- 0.05  ,0.22 +/- 0.09  ,0.27 +/- 0.11  ,156 +/- 8
+8              ,2              ,48             ,2355 +/- 0     ,74.25 +/- 4.33 ,0.152 / 0.072  ,0.198*c - 0.376,0.076*c + 0.430,310.61         ,230.61         ,80.29          ,34.64          ,-              ,0.52 +/- 0.02  ,0.73 +/- 0.36  ,0.23 +/- 0.32  ,0.37 +/- 0.29  ,0.43 +/- 0.29  ,169 +/- 10
+Extracted: 302
+Called: 302
+Scored: 302
```

Comment:
This same test file exists three times in the repository - can we remove the duplicates?